app/controllers/components/course/videos_component.rb

```diff
@@ -27,7 +27,7 @@ def main_sidebar_items
         icon: 'video-camera',
         title: settings.title || t('course.video.videos.sidebar_title'),
         weight: 12,
-        path: course_videos_path(current_course),
+        path: course_videos_path(current_course, tab: current_course.video_tabs.first),
```

Comment:
`course.video_tabs.first` is used quite often. Can we abstract it into a separate method?